ote/src/cljs/ote/app/controller/route.cljs

```diff
@@ -229,7 +230,12 @@
   (process-event [{response :response} app]
     (.error js/console "Save route failed:" (pr-str response))
     (assoc app
-           :flash-message-error "Reitin tallennus epäonnistui")))
+      :flash-message-error "Reitin tallennus epäonnistui"))
```

Comment:
Add text to resource file.

Reply:
Yes, later.
I think that we can use in this poc only finnish and translate all text at once.